integrations/amazon-bedrock.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: integration
+name: Amazon Bedrock
+description: Use Models from AI21 Labs, Anthropic, Cohere, Meta, Stability AI, and Amazon via Amazon Bedrock with Haystack
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/amazon-bedrock-haystack
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/amazon_bedrock
+type: Model Provider
+report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
+logo: /logos/aws.png
+version: Haystack 2.0
+toc: true
+---
+
+### Table of Contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [AmazonBedrockGenerator](#AmazonBedrockGenerator)
+
+## Overview
+
+[Amazon Bedrock](https://aws.amazon.com/bedrock/) is a fully managed service that makes high-performing foundation models from leading AI startups and Amazon available for your use through a unified API.  You can choose from various foundation models to find the one best suited for your use case. More information can be found on the [documentation page](https://docs.haystack.deepset.ai/v2.0/docs/amazonbedrockgenerator).
+
+## Installation
+
+Install the Amazon Bedrock integration:
+```bash
+pip install amazon-bedrock-haystack
+```
+
+## Usage
+
+Once installed, you will have access to an AmazonBedrockGenerator that supports models from various providers: 
+- Anthropic's Claude
+- AI21 Labs' Jurassic-2
+- Stability AI's Stable Diffusion
+- Cohere's Command and Embed
+- Meta's Llama 2
+- Amazon Titan language and embeddings models
```

Comment:
Do we support embedding models? Don't we need another component for them?

Reply:
Embedding models are indeed not supported yet. I created an issue to track that: https://github.com/deepset-ai/haystack-core-integrations/issues/181